test/pred-sets/ref/ccb_quad.inv

```diff
@@ -30,6 +30,9 @@ sd::oec.weighted_labeled_examples 8
 current_pass 1
 l1_state 0
 l2_state 1
+gd_ppw_state.size() = 1
+gd_ppw_state[0]normalized_sum_norm_x = 305.4242420196533
```

Comment:
right now this info is duplicated to maintain compat with previous versions see:

norm normalizer 305.424
total_weight 15.2712

We could skip duplicating this info when ppw_state.size() is equal to 1